pkg/mc/mc.go

```diff
@@ -221,16 +225,21 @@ func (mc *MC) listContexts(cmd Cmd) (contexts []string, err error) {
 }
 
 // do executes a command against kubectl and sends a bool to the done channel when done
-func do(done chan bool, context string, output map[string]json.RawMessage, writeToStdout bool, out io.Writer, cmd Cmd, mutex *sync.Mutex) {
+func do(done chan bool, context string, namespace string, output map[string]json.RawMessage, writeToStdout bool, out io.Writer, cmd Cmd, mutex *sync.Mutex) {
 	stdout, err := kubectl(cmd)
 	if err != nil {
 		stdout = []byte(err.Error())
 	}
 	mutex.Lock()
-	output[context] = stdout
+
+	cns := context
+	if namespace != "" {
+		cns += " " + namespace
+	}
+	output[cns] = stdout
```

Comment:
hmm would the following map be better?
```
context:
  namespace:
    <data>
```

Reply:
I agree it is cleaner, though there's also the case where a namespace is _not_ provided. How do we handle that in a nested map?

Reply:
I guess it would work if we changed the `output` map into a `map[string]interface{}` which would allow us to add multiple levels.
But I don't want to hold up this PR any further and if ever needed we can change it. For let's leave it as `map[string]json.RawMessage`.